scalanativelib/api/src/ScalaNativeWorkerApi.scala

```diff
@@ -50,8 +50,10 @@ object NativeLogLevel {
 sealed abstract class ReleaseMode(val name: String)
 
 object ReleaseMode {
-  case object Debug extends ReleaseMode("debug")
-  case object Release extends ReleaseMode("release")
+  case object Debug extends ReleaseMode("debug")                // fast compile, little optimization
+  case object Release extends ReleaseMode("release")            // same as ReleaseFull for versions 0.3.x
+  case object ReleaseFast extends ReleaseMode("release-fast")   // runtime optimize, faster compile, smaller binary
+  case object ReleaseFull extends ReleaseMode("release-full")   // runtime optimize, prefer speed over compile time and size
```

Comment:
```suggestion
  /** Fast compile, little optimization. */
  case object Debug extends ReleaseMode("debug")
  /** Same as [[ReleaseFull]] for versions 0.3.x. Deprecated since ScalaNative 0.4. */
  case object Release extends ReleaseMode("release")
  /** Runtime optimize, faster compile, smaller binary. */
  case object ReleaseFast extends ReleaseMode("release-fast")
  /** Runtime optimize, prefer speed over compile time and size. */
  case object ReleaseFull extends ReleaseMode("release-full")
```

I had rather some ScalaDoc in mind, which is part of the API and most probably will guide the `build.sc` writer when editing in an IDE.